### PR TITLE
Convert Business verification reducer to typescript

### PR DESCRIPTION
**Pull request by gtaschuk**
**Describe the Pull Request**

Convert Business Verification reducers to typescript (almost done with these)

**Todo**

- [ ] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [ ] Request review from relevant @person or team
- [ ] QA your pull request. This includes running the app, login, testing changes etc.
- [ ] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

File: app/client/components/funding/wyreBankFunding.jsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import {
 } from "../../reducers/wyreReducer";
 import LoadingSpinner from "../loadingSpinner.jsx";
 import AsyncButton from "../AsyncButton.jsx";
-import { loadBusinessProfile } from "../../reducers/businessVerificationReducer";
+import { loadBusinessProfile } from "../../reducers/businessVerification/reducers";
```
Comment (enjeyw) [inline app/client/components/funding/wyreBankFunding.jsx:12]: lol we should probably rename the /reducers/ directory something like /redux/ so we don't have `/reducers/foo/reducers`

Comment (tristanhcole) [inline app/client/components/funding/wyreBankFunding.jsx:12]: this should be an action

File: app/client/components/pages/businessVerificationPage.jsx [hunk 2]
```diff
@@ -15,7 +15,7 @@ import { LightTheme } from "../theme.js";
 import {
   loadBusinessProfile,
   UPDATE_ACTIVE_STEP
-} from "../../reducers/businessVerificationReducer";
+} from "../../reducers/businessVerification/reducers";
```
Comment (tristanhcole) [inline app/client/components/pages/businessVerificationPage.jsx:18]: action

File: app/client/components/verification/businessBankLocation.jsx [hunk 3]
```diff
@@ -11,7 +11,7 @@ import AsyncButton from "../AsyncButton.jsx";
 import {
   UPDATE_ACTIVE_STEP,
   UPDATE_BUSINESS_VERIFICATION_STATE
-} from "../../reducers/businessVerificationReducer";
+} from "../../reducers/businessVerification/reducers";
```
Comment (tristanhcole) [inline app/client/components/verification/businessBankLocation.jsx:14]: consider changing to an action

File: app/client/components/verification/businessDetails.jsx [hunk 4]
```diff
@@ -9,7 +9,7 @@ import {
   editBusinessProfile,
   createBusinessProfile,
   UPDATE_ACTIVE_STEP
-} from "../../reducers/businessVerificationReducer";
+} from "../../reducers/businessVerification/reducers";
```
Comment (tristanhcole) [inline app/client/components/verification/businessDetails.jsx:12]: action

File: app/client/components/verification/businessDocuments.jsx [hunk 5]
```diff
@@ -6,7 +6,7 @@ import {
   editBusinessProfile,
   UPDATE_ACTIVE_STEP,
   uploadDocument
-} from "../../reducers/businessVerificationReducer";
+} from "../../reducers/businessVerification/reducers";
```
Comment (tristanhcole) [inline app/client/components/verification/businessDocuments.jsx:9]: action

File: app/client/sagas/businessVerificationSaga.js [hunk 6]
```diff
@@ -21,7 +23,7 @@ import {
   EDIT_BANK_ACCOUNT_REQUEST,
   EDIT_BANK_ACCOUNT_SUCCESS,
   EDIT_BANK_ACCOUNT_FAILURE
-} from "../reducers/businessVerificationReducer";
+} from "../reducers/businessVerification/reducers";
```
Comment (tristanhcole) [inline app/client/sagas/businessVerificationSaga.js:26]: action

**Pull request merged**